### PR TITLE
Added options.event to set which hook to run limit checks on

### PR DESCRIPTION
**Pull request by moejay**
Related to #5

### CODE REVIEW

File: src/yaral.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const Limitus = require('limitus');
 import { Bucket } from './bucket';
 import { tooManyRequests, Output } from 'boom';
-import { Server, Request, Response, PluginFunction } from 'hapi';
+import {Server, Request, Response, PluginFunction, ServerRequestExtPoints} from 'hapi';
```
Comment (SimonSchick) [inline src/yaral.ts]: Please leave the spacing.

Comment (connor4312) [reply src/yaral.ts]: I'll probably throw prettier into this project after this PR

Comment (moejay) [reply src/yaral.ts]: That would be helpful 😄

**Comment by SimonSchick** [comment]
@connor4312 can you run a release?

**Comment by connor4312** [comment]
Published in 2.1.0

**Pull request merged**